Modules/Express/Request.xojo_code

```diff
@@ -57,43 +63,36 @@ Inherits SSLSocket
 		    
 		  End If
 		  
-		  // Get the request data.
-		  DataGet
-		  
-		  // Get the body from the data.
-		  BodyGet
 		  
 		  // Get the length of the content that has been received.
-		  Var contentReceivedLength As Integer = Body.Bytes
+		  Var DataReceivedLength As Integer = Lookahead.Bytes
```

Comment:
Done... see commit below.

- Data is now always being appended while `.ReadAll`.
- Comparison against `Content-Length` now always compares to `Body.Bytes`
- `Data` is cleared when no longer needed in Method `Process`
- `Process` checks `Body.Bytes <> ContentLength` and returns Error 400 (Bad Request) if needed

So back to previous Body/Content-Length comparison, while still having the fix regarding parsing the Body in case of Multipart Form Data.